Newsletters: add 330 (2024-11-22)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Follow up on unwritten news items @harding 
- [x] Client/services section @bitschmidty 
- [x] Notable merges section @Gustavojfe

harding commented:
Apologies for opening the PR late this week.

LarryRuane commented on _posts/en/newsletters/2024-11-22-newsletter.md:
```diff
@@ -0,0 +1,145 @@
+- **Signet activity report:** Anthony Towns [posted][towns signet] to
+  Delving Bitcoin a summary of activity on the default [signet][topic
+  signet] related to proposed soft forks available through [Bitcoin
+  Inquisition][bitcoin inquisition repo].  The post looks at use of
```
```suggestion
  Inquisition][bitcoin inquisition repo].  The post looks at
```
Or, "the post looks at the use of  [SIGHASH_ANYPREVOUT][topic sighash_anyprevout], including tests ..."

(In other words, remove "usage")

LarryRuane commented on _posts/en/newsletters/2024-11-22-newsletter.md:
```diff
@@ -0,0 +1,145 @@
+  [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify].  It then looks
+  at `OP_CHECKTEMPLATEVERIFY` usage directly, including what are likely
+  several different [vault][topic vaults] constructions and a few data
+  carrier transactions.  Finally, the post looks at usage of
+  [OP_CAT][topic op_cat], including for a proof-of-work faucet (see
```
```suggestion
  carrier transactions.  Finally, the post looks at
  [OP_CAT][topic op_cat] usage, including for a proof-of-work faucet (see
```

Gustavojfe commented:
I just added the merge summaries. I removed [LDK #3283][], since it was included in last week's newsletter.

vostrnad commented on _posts/en/newsletters/2024-11-22-newsletter.md:
```diff
@@ -0,0 +1,191 @@
+  in his design in order to allow it the specification change to be
+  adopted as fast as possible.
+
+- **Signet activity report:** Anthony Towns [posted][towns signet] to
+  Delving Bitcoin a summary of activity on the default [signet][topic
+  signet] related to proposed soft forks available through [Bitcoin
+  Inquisition][bitcoin inquisition repo].  The post looks at use of
+  [SIGHASH_ANYPREVOUT][topic sighash_anyprevout] usage, including tests
+  of [LN-Symmetry][topic eltoo] and emulation of
+  [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify].  It then looks
+  at `OP_CHECKTEMPLATEVERIFY` usage directly, including what are likely
+  several different [vault][topic vaults] constructions and a few data
+  carrier transactions.  Finally, the post looks at usage of
+  [OP_CAT][topic op_cat], including for a proof-of-work faucet (see
+  [Newsletter #306][news306 powfaucet]), a possible vault or other
+  [covenant][topic covenants], and verification of a [STARK][]
+  zero-knowledge proof.
```
Shameless plug: worth mentioning? https://delvingbitcoin.org/t/ctv-apo-cat-activity-on-signet/1257/4

vostrnad commented on _posts/en/newsletters/2024-11-22-newsletter.md:
```diff
@@ -0,0 +1,191 @@
+  process at the terminal, so this change is particularly useful for automated
+  installations.
+
+- [[Core Lightning #7604][] adds the `bkpr-editdescriptionbypaymentid` and
```
```suggestion
- [Core Lightning #7604][] adds the `bkpr-editdescriptionbypaymentid` and
```

bitschmidty commented on _posts/en/newsletters/2024-11-22-newsletter.md:
```diff
@@ -0,0 +1,210 @@
+  opcode allows making a hash commitment to a pair of elements; this is
+  similar to what could be achieved using the proposed [OP_CAT][topic
+  op_cat] concatenation opcode or streaming-SHA opcodes such as those
+  [available][streaming sha] in the Elements [sidechain][topic
```
```suggestion
  [available][streaming sha] in Elements-based [sidechains][topic
```

bitschmidty commented on _posts/en/newsletters/2024-11-22-newsletter.md:
```diff
@@ -0,0 +1,210 @@
+  benefit the extension of protocols such as LN, [Ark][topic ark], [timeout
+  trees][topic timeout trees], [BitVM2][topic acc], and others. This update
+  builds on existing features such as 1P1C relays, [TRUC][topic v3 transaction
+  relay] transactions, and [sibling eviction][topic kindred rbf]. See Newsletter
+  [#328][news328 ephemeral].
```
```suggestion
  relay] transactions, and [sibling eviction][topic kindred rbf] (see Newsletter
  [#328][news328 ephemeral]).
```

bitschmidty commented on _posts/en/newsletters/2024-11-22-newsletter.md:
```diff
@@ -0,0 +1,210 @@
+
+- [Core Lightning #7833][] enables the [offers][topic offers] protocol by
+  default, removing its previous experimental status. This follows the merging
+  of its PR into the BOLTs repository (See Newsletter [#323][news323 offers]).
```
```suggestion
  of its PR into the BOLTs repository (see Newsletter [#323][news323 offers]).
```

bitschmidty commented on _posts/en/newsletters/2024-11-22-newsletter.md:
```diff
@@ -0,0 +1,210 @@
+
+- [Core Lightning #7799][] introduces the `xpay` plugin to send payments by
+  constructing optimal [multipath payments][topic multipath payments], using the
+  `askrene` plugin (See Newsletter [#316][news316 askrene]) and the
```
```suggestion
  `askrene` plugin (see Newsletter [#316][news316 askrene]) and the
```

harding commented:
Made edits for all feedback (thanks @LarryRuane @vostrnad @bitschmidty !), added lede, removed releases/RCs section (none found), reviewed contributions from @bitschmidty and @Gustavojfe (extra great job this week with the deep linking!), and added topic entries.  I also took steps to follow up on the two news items I didn't get to this week.

Thanks everyone!

azuchi commented on _posts/en/newsletters/2024-11-22-newsletter.md:
```diff
@@ -0,0 +1,213 @@
+  wallet that uses Bitcoin Core and coordinates [PSBTs][topic psbt] over nostr.
+
+- **bitcoinutils.dev launches:**
+  The [bitcoinutils.dev][] website provides a variety of Bitcoin utilites
```
```suggestion
  The [bitcoinutils.dev][] website provides a variety of Bitcoin utilities
```

harding replied:
Thanks!

vostrnad commented on _posts/en/newsletters/2024-11-22-newsletter.md:
```diff
@@ -0,0 +1,213 @@
+  zero-knowledge proof.
+
+  Vojtěch Strnad [replied][strnad i.o] that he was inspired by Towns's
+  post to create a [block explorer][topic block explorers] that lists
```
Is it really a "block explorer" when it doesn't actually allow exploring blocks and only shows a subset of transactions? I'd slightly prefer just "explorer" or even "website":
```suggestion
  post to create a website that lists
```

harding replied:
Updated.

bitschmidty reviewed (commented):
ACK lede, topics with couple changes: Reworded some topics to include the block explorer->website wording
